### ISSUE TITLE
IE11: HTML Audio: Sound: Volume is not affected by global game.sound.volume.

### ISSUE THREAD

**Issue by micsun-al** [photonstorm/phaser-ce#617]
This Issue is about (pick one, ✏️ delete others)

* A bug in the API:
  * Phaser version(s): 2.11.1
  * Live example: https://playcode.io/241379
  * What steps produce the bug: On Internet Explorer 11, do "game.sound.volume = 0".
  * What should happen: "game.sound.volume = 0" should make all audio inaudible.
  * What happens instead: Playing audio is audible.

rich seems to know about a fix about multiplying Sound's "volume" property by "_globalVolume", but I was unable to get this to work.

Is it this line?
```
this._sound.volume = this._volume
```
...changing it to something like this?
```
this._sound.volume = this._volume * this._globalVolume
```

Thank you!

**Comment by micsun-al** [comment]
`game.sound.volume` does not seem to affect the volume of the sounds that are playing before or after `game.sound.volume` is changed.

**Comment by micsun-al** [comment]
Here's the chat log from Slack #phaser-ce.

![image](https://user-images.githubusercontent.com/15949921/52025219-752d5b80-24b8-11e9-85ce-d6e4d8ba4b9b.png)

**Comment by micsun-al** [comment]
Tried the change https://github.com/photonstorm/phaser-ce/commit/f44bbb5980c0a38e10492cd7ab5e8d4d7e78857c, and that seems to affect the volume of a currently playing sound (background music), but new sounds don't seem to be affected.  Also, switching to a different window, then back to IE11 seems to continue any audio that was previously affected by the `global.sound.volume` are no longer affected by it (they play at their full volume instead of 0 volume, when `game.sound.volume` is set to 0).

**Comment by samme** [comment]
I'll make a test.

**Comment by samme** [comment]
<https://424v2y53o7.codesandbox.io>

Seems to work.

1. Set **Sound Manager: volume** to 0 in GUI panel.
2. Click **Sound: loopFull** in GUI panel.
3. Wait.
4. Set **Sound Manager: volume** above 0.

**Comment by micsun-al** [comment]
Confirming fix works on Windows 10 (1809) IE11 (11.253.17763.0).

**Comment by samme** [comment]
Fixed in f44bbb5
